Makefile

```diff
@@ -129,6 +129,10 @@ check: $(BIN)
 fmt: $(BIN)
 	@$(BIN) fmt
 
+.PHONY: vendor-lint
+vendor-lint: $(BIN)
+	@$(BIN) exec -- vendor/vendor_updater.exe lint
```

Comment:
As mentioned earlier, these checks should be defined as part of the `runtest` alias.